packages/portal-api/src/ts/error_reporter.ts

```diff
@@ -0,0 +1,37 @@
+import { logUtils } from '@0xproject/utils';
```

Comment:
we now have this same `error_reporter` module in a couple different packages, it might be time to pull it into a shared package

Reply:
Yessss! 🍾